diff -ruN libcst-0.4.10/.cargo/config.toml libcst-0.4.10-patch/.cargo/config.toml
--- libcst-0.4.10/.cargo/config.toml	2023-03-24 04:13:47.000000000 -0600
+++ libcst-0.4.10-patch/.cargo/config.toml	2023-05-25 23:44:24.722756467 -0600
@@ -2,10 +2,12 @@
 rustflags = [
   "-C", "link-arg=-undefined",
   "-C", "link-arg=dynamic_lookup",
+  "-C", "link-arg=-headerpad_max_install_names"
 ]
 
 [target.aarch64-apple-darwin]
 rustflags = [
   "-C", "link-arg=-undefined",
   "-C", "link-arg=dynamic_lookup",
-]
\ No newline at end of file
+  "-C", "link-arg=-headerpad_max_install_names"
+]
